client/package.json

```diff
@@ -13,41 +13,40 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~12.2.15",
-    "@angular/cdk": "~12.2.13",
-    "@angular/common": "~12.2.15",
-    "@angular/compiler": "~12.2.15",
-    "@angular/core": "~12.2.15",
-    "@angular/flex-layout": "~12.0.0-beta.35",
-    "@angular/forms": "~12.2.15",
-    "@angular/material": "~12.2.13",
-    "@angular/platform-browser": "~12.2.15",
-    "@angular/platform-browser-dynamic": "~12.2.15",
-    "@angular/router": "~12.2.15",
+    "@angular/animations": "~13.2.2",
+    "@angular/cdk": "~13.2.2",
+    "@angular/common": "~13.2.2",
+    "@angular/compiler": "~13.2.2",
+    "@angular/core": "~13.2.2",
+    "@angular/flex-layout": "~13.0.0-beta.38",
+    "@angular/forms": "~13.2.2",
+    "@angular/material": "~13.2.2",
+    "@angular/platform-browser": "~13.2.2",
+    "@angular/platform-browser-dynamic": "~13.2.2",
+    "@angular/router": "~13.2.2",
     "rxjs": "~7.5.4",
     "tslib": "^2.0.0",
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/architect": "^0.1302.0",
-    "@angular-devkit/build-angular": "~12.2.15",
-    "@angular-eslint/builder": "~12.7.0",
-    "@angular-eslint/eslint-plugin": "~12.7.0",
-    "@angular-eslint/eslint-plugin-template": "~12.7.0",
-    "@angular-eslint/schematics": "~12.7.0",
-    "@angular-eslint/template-parser": "~12.7.0",
-    "@angular/cli": "~12.2.15",
-    "@angular/compiler-cli": "~12.2.15",
-    "@angular/language-service": "~12.2.15",
-    "@nrwl/cypress": "^13.7.1",
-    "@nrwl/workspace": "^13.5.1",
+    "@angular-devkit/architect": "^0.1302.3",
+    "@angular-devkit/build-angular": "~13.2.3",
+    "@angular-eslint/builder": "~13.1.0",
+    "@angular-eslint/eslint-plugin": "~13.1.0",
+    "@angular-eslint/eslint-plugin-template": "~13.1.0",
+    "@angular-eslint/schematics": "~13.1.0",
+    "@angular-eslint/template-parser": "~13.1.0",
+    "@angular/cli": "~13.2.3",
+    "@angular/compiler-cli": "~13.2.2",
+    "@angular/language-service": "~13.2.2",
+    "@cypress/schematic": "^1.6.0",
     "@types/jasmine": "^3.8.2",
-    "@types/node": "^16.11.22",
-    "@typescript-eslint/eslint-plugin": "^5.10.2",
-    "@typescript-eslint/parser": "^5.12.0",
+    "@types/node": "^17.0.18",
+    "@typescript-eslint/eslint-plugin": "5.12.0",
+    "@typescript-eslint/parser": "5.12.0",
     "cypress": "^9.4.1",
```

Comment:
Interestingly when you add the new `@cypress/schematic` with `ng add` it sets this version to `latest` rather than any specific version.